input/postgres/relation_stats_ext.go

```diff
@@ -51,3 +51,80 @@ func handleRelationStatsExt(db *sql.DB, relStats state.PostgresRelationStatsMap,
 
 	return relStats, nil
 }
+
+const citusIndexSizeSQL = `
+WITH dist_idx_shard_stats_raw AS (
+	SELECT
+		dp.logicalrelid::regclass,
+		(pg_catalog.run_command_on_shards(logicalrelid::regclass::text, $$
+			SELECT
+				jsonb_agg(shard_idx_stats)
+			FROM (
+				SELECT
+					relnamespace::regnamespace AS idx_shard_schema,
+					indexrelid::regclass AS idx_shard_name,
+					COALESCE(pg_catalog.pg_relation_size(indexrelid), 0) AS idx_shard_bytes
```

Comment:
Gathering other distributed stats here doesn't really make sense, right? They're not additive like size, and an average seems dubious as well...

Reply:
Yeah, good question. For indexes specifically, it could be interesting to know "was this index scanned on any of the shards" to drive the unused index recommendations. Maybe we can also just aggregate that across the nodes? (i.e. `pg_stat_user_indexes.idx_scan`, and then we do our usual diffing logic on it)

As you note, averaging could tell the wrong story, so I wouldn't do other metrics (like cache hit ratio).

In the future, we may also want to collect some extra metrics for tables in the same style (e.g. total number of live rows across all shards), but I'd look at that on another day.

Reply:
>For indexes specifically, it could be interesting to know "was this index scanned on any of the shards" to drive the unused index recommendations. Maybe we can also just aggregate that across the nodes?

I thought I had asked this, but it looks like I never submitted my comment or it got lost: "aggregate" as in sum? Or average?

Reply:
FWIW, I mean aggregate on that :)